umh.docs.umh.app/content/en/docs/features/data-connectivity-node-red/index.md

```diff
@@ -18,16 +24,27 @@ By default, you can
 - connect to HTTP using the HTTP node
 
 By installing additional packages, you can:
-- connect to OPC-UA (node-red-controb-opcua)
-- ...
+- connect to OPC-UA (node-red-contrib-opcua)
+- connect to kafka (node-red-contrib-kafkajs)
+- connect to Siemens-S7 (node-red-contrib-s7)
+- connect to serial (node-red-node-serialport)
+
+You can additionally contextualize the data, using function or other different nodes do manipulate the 
+received data.
 
 ## How can I use it?
 
-See also tutorial on accessing Node-RED
+Node-red comes preinstalled as a microservice with the umh. Follow our [get started](/docs/getstarted/) guide, 
+to connect with node-red and create your first flow. We also have many [best practises & guides for Node-RED](https://learn.umh.app/topic/node-red/).
 
 ## What are the limitations?
 
-- most packages have no enterprise support. If you encounter any errors, you need to ask the community. However, we found that these packages are often more stable than the commercial ones out there, as they have been battle tested by way more users than commercial software
-- 
+- most packages have no enterprise support. If you encounter any errors, you need to ask the community. 
```

Comment:
one additional limitation is "the speed of development of Node-RED". Klaus Landsdorf for example spends at least around 40h per year just to keep a node available and up to date. There are half-year significant updates to Node-RED, which might break old nodes. These update cycles put pressure on the IT / OT organization using this as they need to keep Node-RED updated